### DISCUSSION TITLE
Duplicated strings in SystemFonts cache

### DISCUSSION THREAD

**Discussion by theolivenbaum** [SixLabors/Fonts#221] [General] [answered]
Hi @JimBobSquarePants 

Was doing some profiling and I noticed that using `SystemFonts.Find(...)` will result in a lot of  "License" strings from the fonts being loaded in memory.

![image](https://user-images.githubusercontent.com/8791811/143288315-945a9f3d-179a-48ae-bb56-267cf7a79804.png)

![image](https://user-images.githubusercontent.com/8791811/143288064-4e47cbc9-5b10-439d-bc22-b128fff5e4da.png)

![image](https://user-images.githubusercontent.com/8791811/143288136-ec310307-9360-4583-8274-07752c6104d6.png)

I only ever used one font (Arial) but nevertheless it seems like all system fonts are cached in memory.

Is there a way to disable loading this info or loading fonts other than the one I asked for?

**Comment by theolivenbaum** [comment]
In total, there are ~3MB of strings loaded in the font cache:
![image](https://user-images.githubusercontent.com/8791811/143288874-8c8134f6-f5e5-4806-8a67-d3e9fd5c5381.png)

**Comment by tocsoft** [comment, marked as answer]
You can always skip using the system fonts collection and instead create your own FontCollection and load the specific fonts you want loaded yourself to avoid the loading of System wide font metadata. If you never touch the type then the font descriptions will never be loaded.

**Reply by theolivenbaum** [reply]
Yeah was reading the implementation, seems like the easiest way forward! Thanks for the suggestion
